src/Resume_Parser.py

```diff
@@ -0,0 +1,44 @@
+from pyresparser import ResumeParser
+
+import nltk
+
+nltk.download('words')
+nltk.download('stopwords')
+
+
+def parse_cv(path_to_cv):
+    """
+    Extracts data from a PDF resume file.
+
+    Parameters:
+    path_to_cv (str): The path to the PDF resume file.
+
+    Returns:
+    dict: A dictionary containing extracted data, including name, email, phone number, skills, work experience, education, and more.
```

Comment:
It can return None if there is an Exception

Reply:
Done